src/components/SendAsset/SendAsset.js

```diff
@@ -279,7 +243,7 @@ const SendAsset = ({
   };
 
   const handleFormSubmit = async () => {
-    if (submitPressed || !txFeeInfo || !selectedContact || !assetData) return;
+    if (submitPressed || !feeInfo || !selectedContact || !assetData) return;
```

Comment:
I think this also results in the user tapping but no feedback / message in this condition?

Reply:
Added a Toast and error log 👍